liquibase-core/src/main/java/liquibase/util/StringUtil.java

```diff
@@ -1031,6 +1031,14 @@ public static byte[] getBytesWithEncoding(String string) {
         return string.getBytes(StandardCharsets.UTF_8);
     }
 
+    /**
+     * @param value string to process
+     * @return string without any whitespaces formatted to lowercase.
+     */
+    public static String toLowerWithoutWhitespaces(String value) {
```

Comment:
This should be updated to return null if value is null instead of throwing a nullpointerexception

Reply:
Definitely, gonna do that